src/vs/workbench/contrib/debug/common/debugSchemas.ts

```diff
@@ -67,6 +67,11 @@ export const debuggersExtPoint = extensionsRegistry.ExtensionsRegistry.registerE
 					description: nls.localize('vscode.extension.contributes.debuggers.configurationAttributes', "JSON schema configurations for validating \'launch.json\'."),
 					type: 'object'
 				},
+				when: {
```

Comment:
I would love if we added some examples in the description, similar to what AlexR did with tasks.
For example:
"Consider using `shellExecutionSupported`, `virtualWorkspace`, `resourceScheme` as appropriate for this"